packages/shell-api/src/api.ts

```diff
@@ -0,0 +1,102 @@
+// TODO: does it make sense to have all this stuff? Don't we just need enough for the top-level API, ie. the globals?
```

Comment:
Just a reminder for myself to remove this TODO comment before merging and after bringing it up with the team.

Reply:
This would all become one big ShellAPI import in monogdb-ts-autocomplete where we set up the db, sh, rs, sp globals so we can reference things like ShellAPI.Database. But I suppose we can hide it by importing it as a type.

Reply:
I added some regression tests to make sure we don't leak ShellAPI or anything on it as a varaible name, so I think that's at least safe.